plugin/pkg/scheduler/algorithm/priorities/BUILD

```diff
@@ -9,16 +9,14 @@ load(
 go_library(
     name = "go_default_library",
     srcs = [
-        "balanced_resource_allocation.go",
         "image_locality.go",
         "interpod_affinity.go",
-        "least_requested.go",
         "metadata.go",
-        "most_requested.go",
         "node_affinity.go",
         "node_label.go",
         "node_prefer_avoid_pods.go",
         "reduce.go",
+        "resource_allocation.go",
```

Comment:
Instead of merging them, I would suggest to keep three files and only extract the common parts to some common place.  

In this way we will not change the view of pkg but can still eliminate dup. cc @bsalamat WDYT?

Reply:
+1
moving the common part to sth like "resource_allocation.go" and leaving the rest in already existing files is I think a good way of doing it.